src/database/queries/getData.js

```diff
@@ -13,17 +13,26 @@ const checkEmail = (email) => {
   const value = [email];
   return connect.query(sql, value);
 };
+
 const getCategories = () => {
   const sql = 'select * from field;';
   return connect.query(sql);
 };
+
 const getSpecalize = (categoryId) => {
   const sql = {
-    text: 'select id , name  from specialization where field_id = $1',
+    text: 'select id, name from specialization where field_id = $1',
     values: [categoryId],
   };
   return connect.query(sql);
 };
+
+const getProposals = (specId) => {
+  const sql = 'select proposal.title, proposal.description, proposal.id, users.firstname, users.lastname, users.photo_url from proposal inner join users on proposal.user_id = users.id where proposal.specalization_id = $1';
```

Comment:
make your query readable like 
```suggestion
 
  const sql =
"select proposal.title, proposal.description, proposal.id, users.firstname, users.lastname, users.photo_url " +
  "from proposal inner join users" +
  "on proposal.user_id = users.id" +
  "where proposal.specalization_id = $1";
  ```

Reply:
@knanahassouna1  done :+1: